### ISSUE TITLE
[FR] Support for secret environment variables

### ISSUE THREAD

**Issue by rayfoss** [stackblitz/webcontainer-core#589]
**Is your feature request related to a problem? Please describe:**
Currently there is no way to pass API keys, encryption secrets and passwords to the start process at runtime.

**Describe the solution you'd like:**
- A UI for adding secrets, similar to GitLab, the secrets are not copied in other people's forks.
- Alternatively, a `.env` file that gets loaded as env vars for the `npm start` process

**Describe alternatives you've considered:**
Rewriting software to not depend on K8s/GitLab/CodeSandbox process.env secrets. All solutions involve cookies that I like to delete at the first sign of an issue, and they would make the environment variable... reviving the "it works on my computer" problem.

**Comment by rayfoss** [comment]
duplicate of https://github.com/stackblitz/core/issues/1492
